src/app/pages/choose-account/choose-account.tsx

```diff
@@ -34,12 +37,16 @@ export const ChooseAccount = memo(() => {
           {url && <RequesterFlag requester={url.toString()} />}
           <LogomarkIcon width="248px" height="58px" />
           <Stack gap="space.04">
-            <styled.h1 textStyle="heading.05">Choose an account to connect</styled.h1>
+            <styled.h1 textStyle="heading.05">
```

Comment:
I added this code as if you only have BTC connected on Ledger and then try to sign in to something it was just showing the logo and an empty screen

Now it shows:
![Screenshot 2024-04-23 at 11 14 46](https://github.com/leather-wallet/extension/assets/2938440/b902d751-b54d-43e3-ba82-ae70587577b5)

Reply:
I believe we prob need some more explanation of what's going on for users?
something like 'you are connected with ledger in bitcoin mode only, please connect stacks account as well to use this feature' or smth like that

Reply:
@mica000 @markmhendrickson @fabric-8 : 

Do you have any input on what copy to use on this screen? 

Right now :
- AS A USER signed in with ledger
- AND and you only have BTC connected 
- WHEN you try and sign in to a site e.g. Gamma
- THEN you see an empty screen below the leather logo like this 

![Screenshot 2024-04-24 at 11 23 15](https://github.com/leather-wallet/extension/assets/2938440/b919b609-4f8d-4cbd-bd40-67018afa0512)

I've just made a small change so that in these cases we say `No connected accounts found` instead of `Choose an account to connect`

Reply:
Following up here a bit late:

Ideally the user should be able to connect their wallet to any site with a Ledger account, regardless of whether they've connected with Ledger for BTC or STX.

It seems that our legacy endpoint for connecting (used by Gamma and others) is _not_ behaving this way. However, because it's legacy (and deprecated), we don't need to update it per se.

I also assume that our newer endpoint (`getAddresses`) _does_ behave this way, though I haven't tested per se. And as we implement this endpoint for mobile, we should retain this agnostic behavior, allowing users to connect regardless of Ledger connectivity by token.

Reply:
Thanks for following up anyway 👍